lib/serializer/index.ts

```diff
@@ -34,7 +37,8 @@ export function signedTransactionSerializerByProtocolIdentifier(protocolIdentifi
     btc: BitcoinSignedTransactionSerializer,
     // grs: BitcoinSignedTransactionSerializer,
     ae: AeternitySignedTransactionSerializer,
-    xtz: TezosSignedTransactionSerializer
+    xtz: TezosSignedTransactionSerializer,
+    XRP: XrpSignedTransactionSerializer

```

Comment:
why uppercase?

Reply:
> why uppercase?

It somehow did not work for me in the unit tests. Having a second look I think in this case this should just be the same as the protocol Identifier. Small tweak I guess.